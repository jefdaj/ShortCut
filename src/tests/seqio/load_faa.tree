.
|-- cache
|   `-- list
|       |-- 1b0af3628d.txt
|       `-- adfdb6bd8c.txt
|-- exprs
|   |-- list
|   |   |-- 0ec5fc193b.faa.list -> ../../cache/list/1b0af3628d.txt
|   |   |-- 49ae883ffc.faa.list -> ../../cache/list/1b0af3628d.txt
|   |   `-- 9344405238.str.list -> ../../cache/list/adfdb6bd8c.txt
|   |-- load_faa
|   |   `-- dee24d6c3c.faa -> $WORKDIR/data/6803genes.faa
|   |-- str
|   |   `-- ec3b181d8a.str -> ../../cache/list/adfdb6bd8c.txt
|   `-- union
|       `-- 27a934c58e_1c60d8b875.faa.list
`-- vars
    |-- mapped.faa.list -> ../exprs/list/0ec5fc193b.faa.list
    |-- result -> ../exprs/union/27a934c58e_1c60d8b875.faa.list
    `-- single.faa -> ../exprs/load_faa/dee24d6c3c.faa

8 directories, 11 files
