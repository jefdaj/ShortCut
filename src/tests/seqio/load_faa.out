[>BAA10018.2 sll0944 [Synechocystis sp. PCC 6803]
MQCLGKIFLSPGTKPESKNLIVIIFTADRDQRVLALSRIVKRPFFNKNPMPAMSQILDPIPNNQPSALFCCYVNATNQIQ
VARITNVPNWYFERVVFPGQRLVFEAVPSAQLEIHTGMMASSIISDTIPCEQLSIDPDGLAAGGFISPEKEHESEDMTSQ
SLVA
>BAA10019.1 slr0320 [Synechocystis sp. PCC 6803]
...
]
